annotation-file-utilities/src/annotator/find/Criteria.java

```diff
@@ -96,6 +96,14 @@ public boolean isSatisfiedBy(TreePath path) {
    * @return true iff this is the criteria on a receiver
    */
   public boolean isOnReceiver() {
+    ASTPath astPath = getASTPath();
+    if (astPath != null && !astPath.isEmpty()) {
+      ASTPath.ASTEntry entry = astPath.get(-1);
+      return entry.getTreeKind() == Tree.Kind.METHOD
```

Comment:
If there is no ASTPathCriterion, then the result of `getASTPath()` is null, and control passes to the test below.